libparsec/crates/types/tests/unit/local_file_manifest.rs

```diff
@@ -12,81 +12,35 @@ use libparsec_tests_lite::prelude::*;
 
 #[rstest]
 fn serde_local_file_manifest_ok(alice: &Device) {
-    // Generated from Parsec 3.0.0-b.12+dev
+    // Generated from Parsec 3.1.1-a.0+dev
     // Content:
     //   type: 'local_file_manifest'
-    //   base: {
-    //     type: 'file_manifest',
-    //     author: ext(2, 0xde10a11cec0010000000000000000000),
-    //     timestamp: ext(1, 1638618643208821) i.e. 2021-12-04T12:50:43.208821Z,
-    //     id: ext(2, 0x87c6b5fd3b454c94bab51d6af1c6930b),
-    //     parent: ext(2, 0x07748fbf67a646428427865fd730bf3e),
-    //     version: 42,
-    //     created: ext(1, 1638618643208821) i.e. 2021-12-04T12:50:43.208821Z,
-    //     updated: ext(1, 1638618643208821) i.e. 2021-12-04T12:50:43.208821Z,
-    //     size: 700,
-    //     blocksize: 512,
-    //     blocks: [
-    //       {
-    //         id: ext(2, 0xb82954f1138b4d719b7f5bd78915d20f),
-    //         offset: 0,
-    //         size: 512,
-    //         digest: 0x076a27c79e5ace2a3d47f9dd2e83e4ff6ea8872b3c2218f66c92b89b55f36560,
-    //       },
-    //       {
-    //         id: ext(2, 0xd7e3af6a03e1414db0f4682901e9aa4b),
-    //         offset: 512,
-    //         size: 188,
-    //         digest: 0xe37ce3b00a1f15b3de62029972345420b76313a885c6ccc6e3b5547857b3ecc6,
-    //       },
-    //     ],
-    //   }
+    //   base: { type: 'file_manifest', author: ext(2, 0xde10a11cec0010000000000000000000), timestamp: ext(1, 1638618643208821) i.e. 2021-12-04T12:50:43.208821Z, id: ext(2, 0x87c6b5fd3b454c94bab51d6af1c6930b), parent: ext(2, 0x07748fbf67a646428427865fd730bf3e), version: 42, created: ext(1, 1638618643208821) i.e. 2021-12-04T12:50:43.208821Z, updated: ext(1, 1638618643208821) i.e. 2021-12-04T12:50:43.208821Z, size: 700, blocksize: 512, blocks: [ { id: ext(2, 0xb82954f1138b4d719b7f5bd78915d20f), offset: 0, size: 512, digest: 0x076a27c79e5ace2a3d47f9dd2e83e4ff6ea8872b3c2218f66c92b89b55f36560, }, { id: ext(2, 0xd7e3af6a03e1414db0f4682901e9aa4b), offset: 512, size: 188, digest: 0xe37ce3b00a1f15b3de62029972345420b76313a885c6ccc6e3b5547857b3ecc6, }, ], }
```

Comment:
Would be nice to format that part (and the one below) for readability, even if it's not perfect to do it manually.

Reply:
I will revert these changes, see https://github.com/Scille/parsec-cloud/pull/10633/files#r2166307842